packages/restitution/src/HeaderRestitution.tsx

```diff
@@ -12,20 +12,20 @@ type HeaderRestitutionBaseProps = {
   className?: string;
   title:React.ReactNode;
   subtitle?:React.ReactNode;
-  titleLeft?:React.ReactNode;
+  rightTitle?:React.ReactNode;
 }
 
-const HeaderRestitution = ({ title, subtitle, titleLeft, className }: HeaderRestitutionBaseProps) => (
+const HeaderRestitution = ({ title, subtitle, rightTitle, className }: HeaderRestitutionBaseProps) => (
     <header className={className}>
       <div className="af-restitution__header-left">
         <div className="af-restitution__title">
           {title && <span className="af-restitution__title-title">{title}</span>}
           <span className="af-restitution__title-subtitle">{subtitle}</span>
         </div>
       </div>
-      {titleLeft && (<div className="af-restitution__header-right">
+      {rightTitle && (<div className="af-restitution__header-right">
```

Comment:
Should also change the className like af-restitution__title-right

Reply:
It will do a breaking a css/html change Samuel. May be it can be done in another Pr ^^